src/component/Timer/index.jsx

```diff
@@ -30,8 +30,10 @@ const Timer = () => {
       }
       setCurrentPeriod(nextPeriod);
       setTime([nextPeriod.mins, nextPeriod.secs]);
+      if (shouldAutoStart) {
+        setIsRunning(true);
+      }
```

Comment:
@Amna-Qassim I pulled your code locally, removed these 3 lines of code, changed line 25 to `setIsRunning(shouldAutoStart)`, and everything worked as expected. Why did you say that doing this doesn't work for you? Try it out again and if it still doesn't work for you, let me know. If it works, then commit it and push it, then we can merge the PR.

Reply:
I try that but the timer doesn't count well.

Reply:
I try that but the timer doesn't work well. @mujz